src/apps/lwaftr/alarms/lwaftr_default_alarms.inc

```diff
@@ -0,0 +1 @@
+lwaftr-default-alarms.conf
```

Comment:
If it's better coming from a string, that's fine too...  Or, we can make a makefile rule for ".alarms" files, as we did for ".yang" files.